bucket/plex-desktop.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.29.1.1974-a65b3ef3",
+    "description": "Plex for Windows.",
```

Comment:
- How is this different from plex player?
- Dot at the end does not make sense

Reply:
https://www.plex.tv/blog/desktop-af/
"...It has all the glorious player capability of Plex Media Player, plus an exciting new offline capability..."